src/lib/pages/backoffice/Patron/PatronDetails/ItemsSearch/ItemsSearch.js

```diff
@@ -24,8 +24,13 @@ export default class ItemsSearch extends Component {
   };
 
   executeSearch = async (queryString, patronPid) => {
-    const { queryString: propsQueryString, fetchAndCheckoutIfOne } = this.props;
+    const {
```

Comment:
Quickly checking around, as you described, the patronPid is not passed from the search handlers. Can you confirm this and if so, better remove the argument from the `executeSearch` function?

Reply:
The `patronPid` is passed from the `onPasteHandler` function to the `executeSearch` function, but it can be removed and then the `patronDetails` will always be fetched from the props. I've tested this new change and it works the same, should I make this update?

Reply:
I would say so, so we know that information comes from one source only. @kpsherva wdyt?

Reply:
I also don't understand this change well. The `executeSearch` is called as the following:
`await this.executeSearch(queryString, patronDetails.user_pid);`
Why using it via props instead?

From a quick check, the issue looks related to the `onSearchHandler`, which does not pass the `patronPid`. If that is the case, then yes, the signature of the func must be changed and the `patronPid` param should be removed because it will be always `null` onSearch, while it will be correctly set `onPaste`.

Reply:
I would prefer not to risk breaking compatibility now, there are places where the arguments are passed in non explicit way so I prefer not to change the signature

Reply:
The signature is already broken ;) I suspect that when you search, the param `patronPid` is `null`.